docs/ref/checks.txt

```diff
@@ -84,6 +84,14 @@ Django's system checks are organized using the following tags:
 * ``templates``: Checks template related configuration.
 * ``caches``: Checks cache related configuration.
 * ``urls``: Checks URL configuration.
+* ``database``: Checks any database-related connection issues. Database checks
```

Comment:
Shai's wording sounds good to me. Should we rename the similarly name function `check_database_connections` as well?